packages/discovery-react-components/src/components/DocumentPreview/components/HtmlView/HtmlView.tsx

```diff
@@ -18,21 +26,115 @@ interface Props {
   setHideToolbarControls?: (disabled: boolean) => void;
 }
 
-export const HtmlView: FC<Props> = ({ document, setHideToolbarControls, setLoading }) => {
+export const canRenderHtmlView = (document?: QueryResult): boolean => !!get(document, 'html');
+
+const SANITIZE_CONFIG = {
+  ADD_TAGS: ['bbox'],
+  ADD_ATTR: [
+    // bbox
+    'page'
+  ],
+  WHOLE_DOCUMENT: true
+};
+
+const base = `${settings.prefix}--html`;
+
+export const HtmlView: FC<Props> = ({
+  document,
+  highlight,
```

Comment:
what is `highlight` in this context? is it the `table_results` data?

Reply:
Yes.